src/common/string_util.cc

```diff
@@ -377,4 +377,14 @@ std::string EscapeString(std::string_view s) {
   return str;
 }
 
+std::string StringNext(std::string s) {
+  for (auto iter = s.rbegin(); iter != s.rend(); ++iter) {
+    if (*iter != char(0xff)) {
+      (*iter)++;
+      break;
+    }
+  }
+  return s;
```

Comment:
should we append a '0x00' if `s == return s`?

Reply:
I think the semantics is different between append 0x00 and back char plus 1.